Dockerfile

```diff
@@ -30,10 +30,10 @@ COPY ./CMakeLists.txt ../kafka_to_nexus_src/CMakeLists.txt
 
 RUN cd kafka_to_nexus \
     && cmake -DCONAN="MANUAL" --target="kafka-to-nexus" -DCMAKE_BUILD_TYPE=Release -DUSE_GRAYLOG_LOGGER=True -DRUN_DOXYGEN=False -DBUILD_TESTS=False ../kafka_to_nexus_src \
-    && make -j8 kafka-to-nexus \
+    && make kafka-to-nexus \
```

Comment:
Jack found that using `-j8` was not noticeably detrimental to build time on the system test jenkins nodes (which have 2 threads, so we should at least use `-j2`) and significantly improved build time on dev machines.
Given that `ninja` is small and clever enough to figure out how to parallelise itself appropriately, an alternative would be to use that instead of `make`.

Reply:
Fair enough. When running it on my machine I found that single threaded compilation was noticeable faster than `-j8`. If course, I did not test `-j2`.